DemoBoolean.java

```diff
@@ -0,0 +1,11 @@
+package com.edocti.jintro.lab01;
+	
+public class DemoBoolean{
+	public static void main(String[] args){
+		boolean isOld=(age > 65);
```

Comment:
`boolean isOld = age > 65;`
The line above is how I would write the code; you don't need the parenthesis in this case. Also, note how I place the space character: operators (=, +, >, <, etc.) are always between 2 spaces. Of course, this is just styling, but it's good to be pedantic about the code you write.

Now 65 is a magic number. You might want to use it in other parts of the class. Usually these 'magic numbers' are assigned to constants, like this:
`private static final int OLD_AGE = 65;`

This should be outside of the main method and inside of the class. Then in main() you'd have:

`boolean isOld = age > OLD_AGE;`